cooperatives/collectorapp

```diff
@@ -0,0 +1,22 @@
+---
+title: Collector App
+parent: Cooperatives
+---
+
+# Collector App
+
+|                   |                                          |
+|:------------------|:-----------------------------------------|
+| model             | Built Externally, Agencies Contribute
+| service type      | Information Technology
+| country           | United Kingdom
+| government members | Oxfordshire County, Buckinghamshire County
+| government type   | municipal
+| license           | unknwon
+| website           | [https://www.aashtoware.org/](https://www.aashtoware.org/)
+| another website   | [https://www.aashtowareproject.org/](https://www.aashtowareproject.org/)
```

Comment:
I don't think these are the right links. Copy/pasta error? :)

Reply:
Oh gosh, my bad! Fixing right now